code/modules/events/disease_outbreak.dm

```diff
@@ -83,9 +117,49 @@
 	name = "Disease Outbreak: Advanced"
 	typepath = /datum/round_event/disease_outbreak/advanced
 	category = EVENT_CATEGORY_HEALTH
+	weight = 10
 	description = "An 'advanced' disease will infect some members of the crew." //These are the ones that get viro lynched!
 	admin_setup = /datum/event_admin_setup/disease_outbreak/advanced
 
+/datum/event_admin_setup/disease_outbreak/advanced
+	///Admin selected custom severity rating for the event
+	var/chosen_severity
+	///Admin selected custom value for the maximum symptoms this virus should have
+	var/chosen_max_symptoms
+
+/datum/event_admin_setup/disease_outbreak/advanced/prompt_admins()
+	var/candidate_count = candidate_check()
+	if(!candidate_check())
+		tgui_alert(usr, "There are no candidates eligible to recieve a disease!", "Error")
+		return ADMIN_CANCEL_EVENT
+	tgui_alert(usr, "[candidate_count] candidates found!", "Disease Outbreak")
+
+	//Custom virus creation
+	if(tgui_alert(usr,"Customize your virus?", "Glorified Debug Tool", list("Yes", "No")) == "Yes")
+		chosen_severity = tgui_input_list(usr, "Pick a severity!","In the event of an airborne virus, try not to breathe.", list("Medium", "Harmful", "Dangerous", "Cancel"))
+		switch(chosen_severity)
```

Comment:
There's no need for "Cancel", it's just not a great entry to justify having a list-style alert and it's better for it to be a three-choice tgui_alert IMO.

Just have it respect getting a null value back (your code with the switch chain should be fine), just change the type of alert you send to usr here and remove "Cancel". They can close out of the window and have it cancel the event, and you ask if they're happy with their selection at the end anyways (line 156).